analyzers/src/SonarAnalyzer.CSharp/Rules/GetHashCodeEqualsOverride.cs

```diff
@@ -153,11 +139,8 @@ private static bool IfStatementWithSingleReturnTrue(IfStatementSyntax ifStatemen
                 return false;
             }
 
-            return returnStatement.Expression != null &&
-                CSharpEquivalenceChecker.AreEquivalent(returnStatement.Expression, CSharpSyntaxHelper.TrueLiteralExpression);
+            return returnStatement.Expression != null
+                   && CSharpEquivalenceChecker.AreEquivalent(returnStatement.Expression, CSharpSyntaxHelper.TrueLiteralExpression);
```

Comment:
It can be changed to the following return statement and than the previous if can be removed.
```suggestion
            return returnStatement != null
                   && returnStatement.Expression != null
                   && CSharpEquivalenceChecker.AreEquivalent(returnStatement.Expression, CSharpSyntaxHelper.TrueLiteralExpression);
```